From 086df3d92ddebb57ed0e0224008f66de771211ca Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Mon, 17 Oct 2016 11:53:40 -0400
Subject: [PATCH 1291/1722] drm/amdgpu: Remove DAL configs that are no longer
 used

We still check them even though they got removed which leads
to build failures.

Change-Id: Ia3c1560e804473fbf18e669a2fd9ef7dbfb32495
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_device.c | 6 +++---
 drivers/gpu/drm/amd/amdgpu/cik.c           | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
index 4e77d87..3a16fb3 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
@@ -1610,19 +1610,19 @@ static int amdgpu_resume(struct amdgpu_device *adev)
 bool amdgpu_device_asic_has_dal_support(enum amd_asic_type asic_type)
 {
 	switch (asic_type) {
-#if defined(CONFIG_DRM_AMD_DAL) && defined(CONFIG_DRM_AMD_DAL_DCE8_0)
+#if defined(CONFIG_DRM_AMD_DAL)
 	case CHIP_BONAIRE:
 	case CHIP_HAWAII:
 		return amdgpu_dal != 0;
 #endif
-#if defined(CONFIG_DRM_AMD_DAL) && (defined(CONFIG_DRM_AMD_DAL_DCE11_0) || defined(CONFIG_DRM_AMD_DAL_DCE11_2))
+#if defined(CONFIG_DRM_AMD_DAL)
 	case CHIP_CARRIZO:
 	case CHIP_STONEY:
 	case CHIP_POLARIS11:
 	case CHIP_POLARIS10:
 		return amdgpu_dal != 0;
 #endif
-#if defined(CONFIG_DRM_AMD_DAL) && defined(CONFIG_DRM_AMD_DAL_DCE10_0)
+#if defined(CONFIG_DRM_AMD_DAL)
 	case CHIP_TONGA:
 	case CHIP_FIJI:
 		return amdgpu_dal != 0;
diff --git a/drivers/gpu/drm/amd/amdgpu/cik.c b/drivers/gpu/drm/amd/amdgpu/cik.c
index 87b6e3c..c12439c 100644
--- a/drivers/gpu/drm/amd/amdgpu/cik.c
+++ b/drivers/gpu/drm/amd/amdgpu/cik.c
@@ -1910,7 +1910,7 @@ int cik_set_ip_blocks(struct amdgpu_device *adev)
 		amdgpu_ip_block_add(adev, &amdgpu_pp_ip_block);
 		if (adev->enable_virtual_display)
 			amdgpu_ip_block_add(adev, &dce_virtual_ip_block);
-#if defined(CONFIG_DRM_AMD_DAL_DCE8_0)
+#if defined(CONFIG_DRM_AMD_DAL)
 		else if (amdgpu_device_has_dal_support(adev))
 			amdgpu_ip_block_add(adev, &dm_ip_block);
 #endif
@@ -1928,7 +1928,7 @@ int cik_set_ip_blocks(struct amdgpu_device *adev)
 		amdgpu_ip_block_add(adev, &amdgpu_pp_ip_block);
 		if (adev->enable_virtual_display)
 			amdgpu_ip_block_add(adev, &dce_virtual_ip_block);
-#if defined(CONFIG_DRM_AMD_DAL_DCE8_0)
+#if defined(CONFIG_DRM_AMD_DAL)
 		else if (amdgpu_device_has_dal_support(adev))
 			amdgpu_ip_block_add(adev, &dm_ip_block);
 #endif
-- 
2.7.4

